src/Libraries/System.Cloud.DocumentDb.Abstractions/Model/TableInfo.cs

```diff
@@ -61,19 +61,21 @@ namespace System.Cloud.DocumentDb;
     /// <summary>
     /// Gets the table throughput value.
     /// </summary>
-    /// <remarks>
-    /// Default is <see cref="Throughput.Unlimited"/>.
-    /// <seealso cref="Throughput.Value"/>.
-    /// </remarks>
+    /// <value>
+    /// The default is <see cref="Throughput.Unlimited"/>.
+    /// </value>
+    /// <seealso cref="Throughput.Value"/>
     public Throughput Throughput { get; }
 
     /// <summary>
-    /// Gets a value indicating whether a <see cref="ITableLocator"/> required to be used with this table.
+    /// Gets a value indicating whether a <see cref="ITableLocator"/> is required to be used with this table.
     /// </summary>
+    /// <value>
+    /// The default is <see langword="false"/>, which means a locator isn't used even if configured.
+    /// </value>
     /// <remarks>
-    /// Default is <see langword="false"/>, which means locator will not be used even if configured.
-    /// If locator is required, requests will require <see cref="RequestOptions"/> provided to API to provide <see cref="RequestOptions{TDocument}.Document"/>.
-    /// This is the protection mechanism to avoid engineers not designed specific table to forget provide documents when table locator is in use.
+    /// If a locator is required, requests require that <see cref="RequestOptions"/> be specified to provide <see cref="RequestOptions{TDocument}.Document"/>.
+    /// This is a protection mechanism to ensure that the table forgets provided documents.
```

Comment:
@geeknoid I took a stab at rewording it to what I thought it was trying to convey. Not sure if it's correct.